server/routes/routes.js

```diff
@@ -4,5 +4,6 @@ import serverSideRender from '../controllers/serverSideRender'
 const router = express.Router()
 
 router.get('/', serverSideRender)
+router.get('/continenthikes', serverSideRender)
```

Comment:
Can we change the first param to be an array?
i.e `route.get(['/', '/continenthikes'], serverSideRender)`
This allows use to have multiple routes grouped together keeping our code DRY since they all use the `serverSideRender` middleware